KubeArmor/packaging/kubearmor.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=KubeArmor
+
+[Service]
+User=root
+KillMode=process
+EnvironmentFile=/etc/sysconfig/kubearmor.conf
+ExecStart=/usr/local/bin/kubearmor -logPath=/tmp/kubearmor.log -enableKubeArmorHostPolicy=$ENABLE_HOST_POLICY
```

Comment:
If we're going to use kubearmor.conf, can you also add the other options (e.g., -eanbleKubeArmorPolicy, -enableKubeArmorVm) too?